features/admin.connections.v1/components/create/custom-authentication-create-wizard.scss

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+ .wizard-wrapper {
```

Comment:
I hope this class is not meant to be re-used.
If not, Better to scope with a specific class.

```css
.custom-authentication-wizard {
    .wizard-wrapper {
        // rest
}